release.sh

```diff
@@ -107,7 +107,7 @@ echo -e "Making SHA-256 & multi-signature..."
 privKeyFile="$(realpath $1)"
 pubKeyFile="$(realpath $2)"
 cd ${releaseDir}
-sha="$(md5sum ${debFile}|cut -d ' ' -f1)"
+sha="$(sha256sum ${debFile}|cut -d ' ' -f1)"
```

Comment:
Can we do a sha at the protocol level? This assume the system have sha256sum install

Reply:
It's a great advice, however need to provide one more utility. How about leave it as follow-up. (https://github.com/Zilliqa/Issues/issues/185)

Reply:
There is no need to provide a utility. we can use openssl sha. 
https://stackoverflow.com/questions/2262386/generate-sha256-with-openssl-and-c